src/state/forgetPassword/action.js

```diff
@@ -0,0 +1,16 @@
+import { createAsyncThunk } from "@reduxjs/toolkit";
+import { publicPost } from "../../utilities/apiCaller";
+
+export const forgotPassword = createAsyncThunk(
+  "forgotPassword/fetchPassword",
+  async (email, { rejectWithValue }) => {
+    try {
+      const response = await publicPost("forgot-password", email);
+      if (response) {
+        return response.data;
+      }
```

Comment:
We don't need this `if (response)` check. Every request has either `response` or `error`.